apps/consumer-client/src/podExampleConstants.ts

```diff
@@ -43,18 +43,63 @@ export const EXAMPLE_GPC_CONFIG = `{
         },
         "E": {
           "isRevealed": false,
-          "equalsEntry": "pod0.A"
+          "equalsEntry": "podZero.A"
         },
         "owner": {
           "isRevealed": false,
           "isOwnerID": true
         }
       }
+    },
+    "cardPOD": {
+      "entries": {
+        "zupass_title": {
+          "isRevealed": true
+        }
+      }
     }
   },
   "tuples": {
     "tuple0": {
-      "entries": ["pod0.E", "pod0.C"],
+      "entries": ["podZero.E", "podZero.C"],
+      "isMemberOf": "admissiblePairs"
+    }
+  }
+}`;
+
+export const EXAMPLE_GPC_CONFIG2 = `{
+  "pods": {
+    "mainstreamPOD": {
+      "entries": {
+        "A": {
+          "isRevealed": true,
+          "isMemberOf": "admissibleValues"
+        },
+        "C": {
+          "isRevealed": false
+        },
+        "E": {
+          "isRevealed": false,
+          "equalsEntry": "mainstreamPOD.A"
+        },
+        "owner": {
+          "isRevealed": false,
+          "isOwnerID": true
+        }
+      }
+    },
+    "ticketPOD": {
+      "entries": {
+        "ticketID": {
```

Comment:
Where would I get a POD to satisfy this test case?  It would be helpful for quick testing if the examples on the add-pcd page would meet this requirement somehow, so it wouldn't take manually typing POD values to set up a test.